pandas/core/arrays/datetimes.py

```diff
@@ -795,13 +795,14 @@ def month_name(self, locale=None):
 
         Parameters
         ----------
-        locale : string, default None (English locale)
-            locale determining the language in which to return the month name
+        locale : str
+            Locale determining the language in which to return the month name.
+            Default is `None` (English locale).
```

Comment:
Sorry my previous comment was misleading, let me add more detail.

In the first line, after the name of the parameter and the colon, we have the (Python) type or types, and then the default value, and nothing else (e.g. `param_name : int or str, default 0`). In case the default value is `None` and this `None` is just a flag to indicate something, we use instead of `default None` the word `optional`. As a example, if `fillna(value=None)` means that the `None` is the value used to impute, we will still use `default None`. If `None` meant that we won't impute anything, we  would have `optional` instead.

What we need to have in this case is that `None` means that 'English locale' will be used. But anything except the format mentioned before should go in the description. Besides consistency, the reason is that at some point it'd be nice to be able to parse all the information in the line of the type. And if we have explanations there, they'll be a problem.

You have more documentation here: https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-3-parameters